charts/jaeger/templates/query-configmap.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.query.config }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "jaeger.fullname" . }}-ui-configuration
+  labels:
+    {{- include "jaeger.labels" . | nindent 4 }}
+    app.kubernetes.io/component: query
+data:
+  query-ui-config.json: |-
+{{ tpl .Values.query.config . | indent 4 }}
```

Comment:
```suggestion
    {{- tpl .Values.query.config . | nindent 4 }}
```

Reply:
I realize it's not done here either https://github.com/jaegertracing/helm-charts/blob/92de6fd2acb713332b3beb8a1024d68ff942eb3a/charts/jaeger/templates/collector-configmap.yaml#L11, they can both be changed another time.